content/en/service_management/service_level_objectives/_index.md

```diff
@@ -211,21 +213,35 @@ Each event includes the following SLO configuration information:
 
 Three types of SLO audit events appear in the Event Explorer:
```

Comment:
```suggestion
#### Audit event categories

Three types of SLO audit events appear in the Event Explorer:
```

Reply:
Suggesting this change for consistency with the format of the status correction audit events section